---
title: "Data Science in Biology"
subtitle: "CHE00048M"
author: "Emma Rand"
format: revealjs
---

## Who am I?

::: {style="font-size: 80%;"}
-   Computational Biologist

    -   computational reproducibility
    -   omics: stem cells, cancer, development, anaerobic digestion
    -   text analysis to understand scientific literature

-   Lead on the [UKRI Digital Research Skills
    Catalyst](https://www.york.ac.uk/biology/about/news/2025/ukridigitalresearchskillscatalyst/)

-   Director of [Cloud-SPAN](https://cloud-span.york.ac.uk/) which
    trains researchers run 'omics analyses on cloud-based HPC

-   MSc Bioinformatics Programme lead

-   [RForwards](https://forwards.github.io/), member of core team and
    teaching lead
:::

::: {style="font-size: 50%;"}
ORCID: [0000-0002-1358-8275](https://orcid.org/0000-0002-1358-8275)
GitHub: [3mmaRand](https://github.com/3mmaRand) Mastodon:
[\@3mma\@mastodon.social](https://mastodon.social/deck/@3mma)
:::

## Overview

-   Data Science in the biosciences

-   Option module: Bioinformatics Group Project

-   60 credit Research Projects in Biology

# Data Science in the biosciences

## Data Science in the biosciences

Data Science or Bioinformatics?

::: incremental
-   More or less interchangeable

-   Data Science is a broader term

-   Bioinformatics more of a focus on sequencing data

-   because that's what came first!
:::

## Data Science in the biosciences

-   High-throughput sequencing

-   Central Dogma: DNA makes RNA, and RNA makes protein

![The Central
Dogma](images/central-dogma.png){fig-alt="DNA is transcribed to RNA which is translated to proteins"}

-   We can sequence DNA, RNA and protein in bulk, single cell and
    spatially

## Data Science in the biosciences

Typical omics workflow

::: incremental
-   DNA or RNA is extracted from samples

-   It is chopped up into 'small' pieces and sequenced

-   Sequencing is not perfect - some based are "miscalled"

-   The data are files of "reads" i.e., sequences with a quality score

-   Text files e.g., [FASTQ
    format](https://en.wikipedia.org/wiki/FASTQ_format)
:::

## Data Science in the biosciences

Technologies

-   long reads or short reads

::: {style="font-size: 70%;"}
| Platform | Read length | Sequence Accuracy |
|----|----|----|
| Illumina | Short 150 - 300 bp | high |
| PacBio | Long 13,000 - 20,000 bp with a max of 300,000 | low |
| Oxford Nanopore | Long 10,000 - 30,000 bp with a max of 2.3 million | low |
:::

## Typical workflow

::: incremental
-   Quality control of raw reads: trimming and filtering

-   Assembly de novo or align to a reference

-   Counts of reads to measure expression/abundance

-   Normalisation of counts

-   Data visualisation: dimension reduction, clustering

-   Statistical analysis: differential expression/abundance, gene set
    enrichment

-   Data visualisation: Volcano plots, heatmaps, networks
:::

## Data Science in the biosciences

-   Imaging

-   Microscopy including live cyte, MRI, CT, PET, Satellite

-   Mechanistic understanding, Monitoring, classification (e.g.,
    diagnosis) prediction (e.g., prognosis)

## Data Science in the biosciences

-   integrating information from a variety omics, imaging and other
    sources

## Data Science in the biosciences

-   [Kaggle](https://www.kaggle.com/)

# Bioinformatics Group Project

## Bioinformatics Group Project

-   Module co-ordinator: Dr Antje Kuhrs

-   Project Coordinator for Bioinformatics: Dr Daphne Ezer

-   20 credits

-   [Group Project -
    BIO00069M](https://www.york.ac.uk/students/studying/manage/programmes/module-catalogue/module/BIO00069M/latest)

## Bioinformatics Group Project

Module summary

-   You will carry out a research project in a small team

    -   Some work will be carried out collaboratively with other group
        members
    -   while other aspects of the project will be carried out
        individually

-   Project planning, data assembly and analysis

-   Assessment: 4000W Scientific Report

## Bioinformatics Group Project

The project Ara-BOX-cis v2

[Ara-BOX-cis](https://araboxcis.org/) is A G-box regulatory network for
a single cell developmental atlas

-   Work together to produce a revised version

-   Novel single cell RNA-seq data to identify subsets of the gene
    network that are relevant at different developmental stages and cell
    types.

::: {style="font-size: 50%;"}
Daphne Ezer, Samuel J.K. Shepherd, Anna Brestovitsky, Patrick Dickinson,
Sandra Cortijo, Varodom Charoensawan, Mathew S. Box, Surojit Biswas,
Katja E. Jaeger, Philip A. Wigge, The G-Box Transcriptional Regulatory
Code in Arabidopsis, Plant Physiology, Volume 175, Issue 2, October
2017, Pages 628â€“640, <https://doi.org/10.1104/pp.17.01086>
:::

## Bioinformatics Group Project

[background](biology-gp-proj.pdf)

# Research Projects in Biology

## Research Projects in Biology

[Daphne Ezer](https://www.york.ac.uk/biology/our-staff/ezerdaphne/)

**Image Analysis of Seedlings**

::: {style="font-size: 70%;"}
Our lab has several thousand scans of seedlings. Previously, a U-net
based AI tool for measuring the stem (hypocotyl) lengths was published
by another lab. We want to improve the performance of the tool by
training it on our data.
:::

![](images/daphne-ezer.jpg)

## Research Projects in Biology

[Dani Ungar](https://www.york.ac.uk/biology/people/daniel-ungar/)

**Modelling the organisation of glycan building enzymes**

::: {style="font-size: 70%;"}
Glycans are the third main type of biological polymer besides proteins
and DNA/RNA and is made up of sugar building blocks. In contrast to the
other two classes though, glycans are synthesized without a template. We
use a computational model to understand the glycan generating enzyme
organisation from measured glycan distributions.
:::

![](images/daniel-ungar.jpg)

# What questions do you have?
